components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -61,13 +61,19 @@
 
 	boolean pkceMandatory = false;
 	boolean pkceSupportPlain = false;
-	
+	boolean bypassClientCredentials = false;
+
 	if(request.getParameter("pkce") != null) {
 		pkceMandatory = true;
 	}
+
 	if(request.getParameter("pkce_plain") != null) {
 		pkceSupportPlain = true;
 	}
+
+	if (request.getParameter("bypass_client_credentials") != null) {
+        bypassClientCredentials = true;
```

Comment:
Shall we fix formatting issues, line #66, #70, and #75?

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1058